tests/test_templatetags.py

```diff
@@ -1320,8 +1320,8 @@ def test_inner_slot_iteration_nested(self):
         component.registry.register("slot_in_a_loop", self.ComponentSimpleSlotInALoop)
 
         objects = [
-            {"inner": ["OBJECT1_ITER1", "OBJECT2_ITER1"]},
-            {"inner": ["OBJECT1_ITER2", "OBJECT2_ITER2"]},
+            {"inner": ["ITER1_OBJ1", "ITER1_OBJ2"]},
```

Comment:
Just renamed these, so the loop part (`ITER`) is first, so it's easier to skim through.